### ISSUE TITLE
ValueError: Namespace NM not available

### ISSUE THREAD

**Issue by syphernl** [mickenordin/wireguide#1]
I randomly came across this project and decided to try it out. 
Unfortunately out-of-the-box it is not working for me. When I start it, it will throw an exception:

```
Traceback (most recent call last):
  File "/tmp/.mount_WireGUSF3qn3/usr/bin//wireguide", line 16, in <module>
    gi.require_version("NM", "1.0")
  File "/tmp/.mount_WireGUSF3qn3/usr/lib/python3/dist-packages/gi/__init__.py", line 126, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace NM not available
```

This happens with both the AppImage as the pip image.

These are the packages / versions I am using for libnm
```
❯ dpkg -l | grep libnm
ii  libnm0:amd64                               1.26.2-1ubuntu1                            amd64        GObject-based client library for NetworkManager
ii  libnma-common                              1.8.30-1                                   all          NetworkManager GUI library - translations
ii  libnma0:amd64                              1.8.30-1                                   amd64        NetworkManager GUI library
```

```
❯ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.10
DISTRIB_CODENAME=groovy
DISTRIB_DESCRIPTION="Ubuntu 20.10"
```

**Comment by mickenordin** [comment]
I will look in to this in the weekend, when I have time to set up some virtual machines. It works for me on groovy and focal, most likely because I allready have some dependency installed which is not properly packed in the appimage/pip-package.

**Comment by mickenordin** [comment]
I just installed a clean virtual machine with Ubuntu 20.10 desktop and I am unable to reproduce this. I think that it is because the package gir1.2-nm-1.0 is installed by default. Can you check if you have gir1.2-nm-1.0 installed?

If not, try:
```
sudo apt install gir1.2-nm-1.0
```
And see if this fixes the error?

**Comment by syphernl** [comment]
Installing `gir1.2-nm-1.0` fixes the error. It seems like this package wasn't included in 20.04 which I originally used to install with, which I later on upgraded to 20.10.

**Comment by mickenordin** [comment]
@syphernl I have found that pip/appimage is not suitable for this project so instead I just published a debian package. If you would like to try it out and test that it works, I'd be thankfull. It should work on Debian testing and Ubuntu 20.04+.
